src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="scrollable">
+    <ng-container *ngFor="let item of items">
```

Comment:
Remove redundant wrapper:
```suggestion
     <kirby-chip *ngFor="let item of items">
```

Reply:
Template updated